EXAMPLES.md

```diff
@@ -0,0 +1,5 @@
+## Example Apps
+
+* **[Calculator](https://ahfarmer.github.io/calculator/)** This is a full-screen, mobile-friendly calculator app similar to the iOS calculator:  [view code in on the GitHub repository](https://github.com/ahfarmer/calculator).
```

Comment:
Let's reformat as:

* **[ahfarmer/calculator](https://github.com/ahfarmer/calculator)** ([demo](https://ahfarmer.github.io/calculator/)): This is a full-screen, mobile-friendly calculator app similar to the iOS calculator.